draft-ietf-rtcweb-jsep.xml

```diff
@@ -407,6 +407,53 @@ setRemote(ANSWER) |                                   |
         sort of SessionDescription to produce.</t>
       </section>
 
+      <section title="Session Description control" anchor="sec.session-description-ctrl">
+	<t>JSEP defines the following objects as an API to allow the
+	application to control session descriptions: RtpTransceivers,
+	RtpSenders, and RtpReceivers.
+	</t>
+
+	<section title="RtpTransceivers" anchor="sec.rtptransceivers">
+	  <t>RtpTransceivers allow the application to control the RTP
+	  media associated with one m= section.  Each RtpTransceiver
+	  has an RtpSender and an RtpReceiver, described below, which
+	  an application may use to control the sending and receiving
+	  RTP media.  The application may also modify the
+	  RtpTransceiver directly, such as by closing it.</t>
+	  
+	  <t>RtpTransceivers can be created either by the application
+	  explicitly by methods to add them, or implicitly by calling
+	  setRemoteDescription with an offer that adds new m=
+	  sections.</t>
+
+	  <t>When first created, the transceiver may be in a state which
```

Comment:
I think these details should be saved for section 4, where we can introduce these concepts in detail.